### DISCUSSION TITLE
Running Visual Tests on a Single Story

### DISCUSSION THREAD

**Discussion by thafryer** [chromaui/addon-visual-tests#320] [Ideas]
We have gotten a few requests to add the ability to run Visual Tests on a Single Story. Users see running the entire test suite as being counterproductive and uses up their Snapshot quota.

**Comment by shilman** [comment]
@thafryer How do we quantify the demand? I've also had people request this, and personally think it's a great idea, but it's still anecdotal. And we've proposed this before and focused on turbosnap instead.

**Reply by thafryer** [reply]
Yea, I'm not quite sure on how to quantify. We had 2 requests come in via Customer support, though.
